src/pages/storage/storage-new.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <page :title="title" :breadcrumb-links="breadcrumbLinks" :errors="errors">
+    <template #header-right>
+      <button-overlay :show="processing">
+        <b-button variant="primary" v-on:click="onCreateClick">Create</b-button>
+      </button-overlay>
+    </template>
+    <div class="pr-3">
+      <div class="pt-3">
+        <label class="form-label">Hostname</label>
+        <b-form-input v-model="hostName" type="text" :state="hostnameValidation"/>
+      </div>
+
+      <div class="pt-3">
+        <label class="form-label">Port</label>
+        <b-form-input v-model="port" type="number" :state="portValidation"/>
```

Comment:
Let's make this a text field and show a validation error if a non-numeric value is entered. Follow existing validation style.
https://github.com/SciGaP/emcenter-gateway/blob/master/src/pages/group/group-new-page.vue#L11

Reply:
Changing the port field to `text` from `number` possibly creates new validation scenarios and corresponding error messages for the users to understand so I provided the placeholder(which tells the users its number field) keeping the type as `number`.